packages/ui/src/components/Input/Input.tsx

```diff
@@ -171,7 +171,7 @@ function Input({
           ref={inputRef}
           type={type}
           value={reveal && hidden ? HIDDEN_PLACEHOLDER : value}
-          className={cn(inputClasses)}
+          className={inputClasses.join(' ')}
```

Comment:
if (error) inputClasses.push(__styles.variants.error)
  if (!error) inputClasses.push(__styles.variants.standard)
  if (size) inputClasses.push(__styles.size[size])
  if (disabled) inputClasses.push(__styles.disabled)
  if (inputClassName) inputClasses.push(inputClassName)
  if (icon) inputClasses.push(__styles.with_icon)
  
  try moving this to bottom ?